.circleci/config.yml

```
@@ -31,6 +34,11 @@ jobs:
           name: lint
           command: make lint
 
+      - save_cache:
+          key: pre-commit-{{ checksum ".pre-commit-config.yaml" }}
+          paths:
+            - "/root/.cache"
```

Maybe,

```suggestion
            - "/root/.cache/pre-commit"
```

as there are other programs that use `.cache` for a cache directory (pip, playwright, firefox, ...).

---

I want to cache those things too (currently pip and black).